$OpenBSD: patch-tests_unittests_torture_misc_c,v 1.6 2021/08/27 08:57:30 sthen Exp $

Disable unit test "torture_path_expand_tilde_unix": real user required.

Index: tests/unittests/torture_misc.c
--- tests/unittests/torture_misc.c.orig
+++ tests/unittests/torture_misc.c
@@ -666,7 +666,8 @@ int torture_run_tests(void) {
 #ifdef _WIN32
         cmocka_unit_test(torture_path_expand_tilde_win),
 #else
-        cmocka_unit_test(torture_path_expand_tilde_unix),
+	/* failed with our make regress ("env -i"), need a real user to be tested.
+        unit_test(torture_path_expand_tilde_unix), */
 #endif
         cmocka_unit_test_setup_teardown(torture_path_expand_escape, setup, teardown),
         cmocka_unit_test_setup_teardown(torture_path_expand_known_hosts, setup, teardown),
